docs/best-practices/data-import.rst

```diff
@@ -350,15 +364,16 @@ statement using the ``PARTITION`` clause::
   COPY OK, 23 rows affected (... sec)
 
 This way, CrateDB does not need to resolve the partition for each row that is
-imported but can store it directly into the correct place resulting in a much
+imported, but can store it directly into the correct place resulting in a much
 faster import.
 
 However, it's still possible (but not recommended) to import into partitioned
-tables without the ``PARTITION`` clause and have the column value in the source.
+tables without the ``PARTITION`` clause and have the column value in the
+source.
 
 When importing data into a partitioned table with existing partitions, it may
-be wanted to apply import optimizations like e.g. disable the `Refresh
-Interval`_ only to newly created partitions. This can be done by altering the
+be desirable to apply import optimizations, like e.g. to disable the `Refresh
```

Comment:
"like e.g." seems inelegant

Reply:
Yeah this sentence needs a rewrite anyway